gtsfm/loader/olsson_loader.py

```diff
@@ -76,20 +81,25 @@ def __init__(
 
         # stores camera poses (extrinsics) and intrinsics as 3x4 projection matrices
         # 'P' array will have shape (1,num_imgs), and each element will be a (3,4) matrix
-        data = loadmat(cam_matrices_fpath)
+        data = scipy.io.loadmat(cam_matrices_fpath)
+
+        if len(data["P"][0]) != self._num_imgs:
+            raise RuntimeError("Number of images found on disk not equal to number of ground truth images.")
 
         # M = K [R | t]
         # in GTSAM notation, M = K @ cTw
         M_list = [data["P"][0][i] for i in range(self._num_imgs)]
```

Comment:
Can we rename this to `projection_matrices`? For readable imo

Reply:
Sure, renamed now.